ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfig.java

```diff
@@ -135,6 +135,8 @@ static SpecConfigBuilder builder() {
 
   int getProposerScoreBoost();
 
+  boolean isEquivocatingIndicesEnabled();
```

Comment:
hmm, I don't think this belongs in `SpecConfig` as it's not coming from the config the spec provides. For the storage side it should just wind up in `StorageConfiguration`.